lib/google_contacts_api/groups.rb

```diff
@@ -23,5 +23,28 @@ def get_groups(params = {})
       end
       GoogleContactsApi::GroupSet.new(response.body, @api)
     end
+
+    def create_group(title)
```

Comment:
Yeah, it looks like you can have arbitrary "extended properties" for Google groups (https://developers.google.com/google-apps/contacts/v3/#creating_contact_groups) but I didn't notice any other core attributes you can set besides `title` though I may have missed them and giving the ability to add extended attributes to groups could be helpful in the future for some people.